tests/unittest_brain_qt.py

```diff
@@ -28,11 +30,25 @@ def test_value_of_lambda_instance_attrs_is_list():
         from PyQt6 import QtPrintSupport as printsupport
         printsupport.QPrintPreviewDialog.paintRequested  #@
         """
-        AstroidManager.brain["extension_package_whitelist"] = {"PyQt6.QtPrintSupport"}
         node = extract_node(src)
         attribute_node = node.inferred()[0]
         if attribute_node is Uninferable:
             pytest.skip("PyQt6 C bindings may not be installed?")
         assert isinstance(attribute_node, UnboundMethod)
         # scoped_nodes.Lambda.instance_attrs is typed as Dict[str, List[NodeNG]]
         assert isinstance(attribute_node.instance_attrs["connect"][0], FunctionDef)
+
+    @staticmethod
+    def test_implicit_parameters():
+        """Regression test for https://github.com/PyCQA/pylint/issues/6464"""
+        src = """
+        from PyQt6.QtCore import QTimer
+        timer = QTimer()
+        timer.timeout.connect  #@
+        """
+        node = extract_node(src)
+        attribute_node = node.inferred()[0]
+        if attribute_node is Uninferable:
+            pytest.skip("PyQt6 C bindings may not be installed?")
```

Comment:
Don't we want to force the c binding to work correctly in at least one of the CI pipeline ? Otherwise this test can work fine online and fail for someone with QT locally. I think we could create a pytest mark for "qt" and a CI job with a proper installation.

Reply:
Hmm, it means we're going to catch issues at release time right ? When/how are we launching this workflow ?

Reply:
Just manually with a reminder in the release.md.

Installing `qt` [takes 39s](https://github.com/PyCQA/astroid/runs/6024015453?check_suite_focus=true). Unless there's a way to cache it?

Reply:
For 40s we might as well add it for each commit (one interpreter only ?). There's a lot of available runner for astroid and they run in a reasonable time (compared to pylint anyway).

Reply:
Actually, it takes 8s, duh, because most of that linked run was setting up the env!

Reply:
I'm confused - why does "Install Qt" actually only do `sudo apt-get install build-essential libgl1-mesa-dev`? You are installing PyQt from binary wheels, so that 40s step can probably just be dropped entirely.

Reply:
We discovered that the `PyQt` wheel doesn't necessarily install the C bindings on `ubuntu-latest`, see [failing run ](https://github.com/PyCQA/astroid/runs/5893185699?check_suite_focus=true) and [comment](https://github.com/PyCQA/astroid/pull/1505#discussion_r846644697).